### PR TITLE
Add QA workflow details

### PR DESCRIPTION
**Pull request by lindalumitchell**
Adding details to the QA Workflow page. Draft here: https://docs.google.com/document/d/16PRqWumFvbKyZ9siDReJIZHEtXaU2f5pF_e3juvn3Qg

Details include daily, weekly, monthly, and as-needed QA responsibilities. Also extra detail about PR reviews, Jira tickets, and UI test automation.

### CODE REVIEW

File: operations/research-and-development/quality/qa-workflow/README.md [hunk 1]
```diff
@@ -1,3 +1,94 @@
...
+1. SDET rotation to monitor nightly automated tests and investigate and update as needed (see below)
+2. Oversee your team's release testing and QA-approve releases (cadence may vary by team / product)
+3. (QA Platform) Detox mobile release testing and maintenance
+
+
+### As Needed
+1. Develop test plans for new features, add test cases to Zephyr test management
+2. Approve new features after manual testing passes
+3. (QA Platform) Oversee testing infrastructure and server needs
+4. Help tend to community campaigns such as test automation Hackathons
```
Comment (lindy65) [inline operations/research-and-development/quality/qa-workflow/README.md]: Can we add "and QA community feature testing"

Not sure about wording but something along those lines?

**Review by lindy65 — DISMISSED** [review]
Looks good to me - just one comment to perhaps include?

**Comment by lindalumitchell** [comment]
> Looks good to me - just one comment to perhaps include?

Great suggestion; I added it. Thanks!

**Review by lindy65 — DISMISSED** [review]
Awesome :) Thanks 👍

File: operations/research-and-development/quality/qa-workflow/README.md [hunk 2]
```diff
@@ -1,3 +1,94 @@
...
+### Weekly
+1. Participate in team triage / planning meetings
+2. Participate in [QA Guild meeting](https://community.mattermost.com/core/channels/qa-weekly-meetings)
+3. Consider giving a demo in QA meeting and/or R&D meeting
+4. (QA Platform) Rainforest release testing and maintenance
+5. (QA Platform) Cypress webapp release testing and maintenance
+6. (QA Platform) Manual release testing and maintenance
+
+### Monthly
+1. SDET rotation to monitor nightly automated tests and investigate and update as needed (see below)
```
Comment (cwarnermm) [inline operations/research-and-development/quality/qa-workflow/README.md]: Could we spell out this acronym, please?

**Review by cwarnermm — DISMISSED** [review]
Such a great addition to the Handbook! There's an acronym we're using that I wasn'r familiar with -- let's spell that one out for transparency. Approving this excellent update!

File: operations/research-and-development/quality/qa-workflow/README.md (hunk 2 above)
Comment (lindalumitchell) [reply operations/research-and-development/quality/qa-workflow/README.md]: Great feedback, @cwarnermm thanks! Done.

**Comment by lindalumitchell** [comment]
(Oops, I inadvertently did something that "dismissed stale reviews," apparently. Re-requesting.)

**Review by lindy65 — APPROVED** [review]
No worries :)

**Review by cwarnermm — APPROVED** [review]
Thank you!

**Pull request merged**